packages/wasmer/wasmer.1.0.0/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+
+authors: ["rajdakin"]
+maintainer: "engineering@wasmer.io"
+license: "MIT"
+homepage: "https://github.com/wasmerio/wasmer-ocaml"
+doc: "https://github.com/wasmerio/wasmer-ocaml"
+bug-reports: "https://github.com/wasmerio/wasmer-ocaml/issues"
+dev-repo: "git+https://github.com/wasmerio/wasmer-ocaml.git"
+tags: ["org:wasmer"]
+url {
+	src: "https://github.com/wasmerio/wasmer-ocaml/archive/refs/tags/1.0.0.zip"
+	checksum: [] # Placeholder, TODO
+}
+
+synopsis: "The official Wasmer bindings for OCaml"
+
+depends: ["ocaml" {>= "4.14.1"} "ocamlfind" "ctypes" "ctypes-foreign" "libwasmer"]
+build: [[make "all"]]
+run-test: [[make "check"]]
+build-env: [
+	[ECHO = "true"]
+]
+setenv: [ # Hacky, but should work; prioritizes the opam library over the system one
+	[LIBRARY_PATH := "%{libwasmer:lib}%"]
+	[LD_LIBRARY_PATH := "%{libwasmer:lib}%"]
+	[DYLD_LIBRARY_PATH := "%{libwasmer:lib}%"]
```

Comment:
I'm not sure what you are trying to do with that but this is going to be refused. You are resetting the user environment. You might want to use `build-env` instead of `setenv` if this is just to build it.

Reply:
This is indeed used during build (which is why I then added it back to `build-env`), but the library is also required to be available at run time.
Also, I've read that `:=` is supposed to prepend the argument and adding a separator if necessary in the docs (instead of resetting the variable entirely)...
If there is a solution (like installing it by the `libwasmer` package), please tell me.

Reply:
I fixed the issue in release 1.1.0.